snapcraft.yaml

```diff
@@ -195,6 +195,15 @@ parts:
     organize:
       usr/bin/: bin/
       usr/lib/: lib/
+    override-prime: |-
```

Comment:
Does this skips installation of `stage-packages`? As far as I understand `stage-packages` will be tried to install anyways :(

Reply:
Thanks, fixed now